### ISSUE TITLE
Cables and Create schematics (.nbt) burned my creative testing world

### ISSUE THREAD

**Issue by gutteringHearthplace** [refinedmods/refinedstorage#3405]
### Describe the bug

I was using the Create mod's schematicannon when the crash occurred- I've already sent this report to them, too. Including a cable from your mod in a build looks like it crashes the game on loading the schematic into a function designed to produce a block checklist. The crash would recur on opening the world until I deleted the schematic entirely. 

Here's text of the report I sent to them: 

### How can we reproduce this bug or crash?

1. Created a schematic of a bulk processing plant I wanted to be able to use in the future, containing blocks from Create and other mods- most notably Cables and Importers from Refined Storage, though I think one or two Quark blocks and the Tinker's Construct clear item frames were also present.
2. Save the schematic and load it into the Schematicannon to test the reproduced functionality. Everything is fine until I add the book for the checklist, upon which the game crashes.
3. Restart Minecraft, reload my testing world. Minecraft crashes again.

### What Minecraft version is this happening on?

Minecraft 1.18.2

### What Forge version is this happening on?

40.1.48

### What Refined Storage version is this happening on?

1.10.2

### Relevant log output

```shell
https://gist.github.com/gutteringHearthplace/c2be8bc17e8340fbc7c0fa3d280036a0

**Comment by gutteringHearthplace** [comment]
Create bug report for the same issue: 
https://github.com/Creators-of-Create/Create/issues/4001

**Comment by raoulvdberge** [comment]
No need to report this to create, I believe this issue is on us.

**Comment by raoulvdberge** [comment]
I added some extra code to avoid the unsafe ServerLevel casting, but unfortunately it just generates a new crash. No network present at location. RS 1 just isn't compatible. It appears like the block place hook isn't called, leading to RS not creating a network and just crashing. 

Will categorize this under #3424

**Comment by raoulvdberge** [comment]
Hello. This bug has been fixed in v1.12.4 of Refined Storage. If this issue persists anyway, please feel free to create a new bug.

**Comment by raoulvdberge** [comment]
This bug has also been fixed in **v1.11.7** of Refined Storage for **Minecraft 1.19.2**. If this issue persists anyway, please feel free to create a new bug.

**Comment by raoulvdberge** [comment]
Hello, this bug is now also fixed for Minecraft 1.18.2 in v1.10.6.
